iOS_SDK/OneSignalSDK/OneSignalUser/Source/Executors/OSPropertyOperationExecutor.swift

```diff
@@ -255,6 +255,14 @@ class OSPropertyOperationExecutor: OSOperationExecutor {
                     OSBackgroundTaskManager.endBackgroundTask(backgroundTaskIdentifier)
                 }
             }
+            if let onesignalId = request.identityModel.onesignalId {
+                if let rywToken = response?["ryw_token"] as? String
+                {
+                    OSConsistencyManager.shared.setRywToken(id: onesignalId, key: OSIamFetchOffsetKey.user, value: rywToken)
+                } else {
+                    OSConsistencyManager.shared.resolveConditionsWithID(id: OSIamFetchReadyCondition.CONDITIONID)
```

Comment:
If we don't have a onesignalId not resolving here would result in no IAM being shown. I suppose that may never happen though.

Reply:
Hmm the onesignal ID should actually always exist at this point, since you needed the onesignal ID to send this request in the first place. The `if let` is more to appease Swift null checks.